docs/source/reporters.rst

```diff
@@ -290,6 +290,20 @@ public Matrix room, as the messages quickly become noisy:
      minimal: true
      enabled: true
 
+E-Mail via sendmail
+---------------------
+
+You can send email via the system's sendmail command provided by the MTA. You need to set "method: sendmail" in the config file.
+
+.. code:: yaml
+
+    report:
+      email:
+        enabled: true
+        from: 'postmaster@example.com'
```

Comment:
Should probably have a `to:` in this example as well.

Reply:
done